docs/source/upcoming_release_notes/1016-kbo-coating-states.rst

```diff
@@ -0,0 +1,32 @@
+1016 kbo-coating-states
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+- N/A
+
+New Devices
+-----------
+`KBOMirrorHEStates` - a class for KBO mirrors with coating states
+and cooling.
+`KBOMirrorStates` - a class for KBO mirrors with coating states
+and no cooling
+Bugfixes
+--------
+- N/A
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+nrwslac
```

Comment:
The bullet points and newlines are actually functional in rst and shouldn't be removed

```suggestion
- `KBOMirrorHEStates` - a class for KBO mirrors with coating states
  and cooling.
- `KBOMirrorStates` - a class for KBO mirrors with coating states
  and no cooling

Bugfixes
--------
- N/A

Maintenance
-----------
- N/A

Contributors
------------
- nrwslac
```

Reply:
fixed style